analyzer/analyzer.go

```diff
@@ -291,22 +291,22 @@ if (plugins.hasPlugin('com.android.application')) {
 func parseBuildGradle(buildGradleContent string) (ProjectDependenciesModel, error) {
 	platformVersion, err := parsePlatformVersion(buildGradleContent)
 	if err != nil {
-		return ProjectDependenciesModel{}, fmt.Errorf("failed to determine compileSDKVersion, error: %s", err)
```

Comment:
please do not return with capitalised error. The main caller function should handle to print the error message with the correct capital.